omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java

```diff
@@ -0,0 +1,240 @@
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.controller;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.idgen.IdentifierPool;
+import org.openmrs.module.idgen.RemoteIdentifierSource;
+import org.openmrs.module.idgen.SequentialIdentifierGenerator;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class IdentifierSourceRestControllerTest extends MainResourceControllerTest {
+	
+    public static final String SEQUENTIAL_IDENTIFIER_SOURCE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a901";
+    public static final String REMOTE_IDENTIFIER_SOURCE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a902";
+    public static final String POOL_SOURCE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a903";
+    public static final String IDENTIFIER_TYPE_UUID = "2f470aa8-1d73-43b7-81b5-01f0c0dfa53c";
+    public static final String IDENTIFIER_SOURCE_TO_PURGE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a999";
+    public static final String SEQUENTIAL_IDENTIFIER_SOURCE_TYPE = "SequentialIdentifierGenerator";
+    public static final String REMOTE_IDENTIFIER_SOURCE_TYPE = "RemoteIdentifierSource";
+    public static final String IDENTIFIER_POOL_SOURCE_TYPE = "IdentifierPool";
+
+    private IdentifierSourceService service;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSet("org/openmrs/module/idgen/include/TestData.xml");
+        this.service = Context.getService(IdentifierSourceService.class);
+    }
+
+    @Override
+    public String getURI() {
+        return "idgen/identifiersource";
+    }
+
+    @Override
+    public String getUuid() {
+        return SEQUENTIAL_IDENTIFIER_SOURCE_UUID;
+    }
+
+    @Override
+    public long getAllCount()  {
+        return service.getAllIdentifierSources(false).size();
+    }
+
+    @Override
+    @Test(expected = ResourceDoesNotSupportOperationException.class)
+    public void shouldGetAll() throws Exception {
+        super.shouldGetAll();
+    }
+
+    @Test
+    public void shouldGetAnIdentifierSourceByUuid() throws Exception {
+
```

Comment:
Can we also remove the above empty line?

Reply:
fixed